API_DOCUMENTATION.apib

```diff
@@ -104,6 +112,12 @@ Polls is a simple API allowing consumers to view polls and vote in them.
 
 ### Insert [POST]
 
++ Attributes
+    + id (string)
+    + login (string)
```

Comment:
https://github.com/RisingStack/bootcamp-roland/blob/a72d9b113dbc8a9bc053d9bad5f7e28afe7c06c7/db/models/user.js#L5
Please make login required as defined in the validation schemas.